test/mason/publish-tests/publish.good

```diff
@@ -0,0 +1,11 @@
+Created new library project: test_dir
+Publish a package to the mason-registry repository
+
+Usage:
+    mason publish [options] <username>
```

Comment:
I think username will be optional later, so we could make it optional now, e.g. `--username`.